### PR TITLE
#748 [Observability] Integrating the service map in distributed tracing

### PR DESCRIPTION
**Pull request by khanhtrand**

### CODE REVIEW

**Comment by khanhtrand** [comment]
image

File: docker-compose.yml [hunk 1]
```diff
@@ -462,7 +462,7 @@ services:
     networks:
       - yas-network
   postgres:
-    image: debezium/postgres:15-alpine-custom
```
Comment (thiennn) [inline docker-compose.yml:465]: we should undo this change. This should be handled in other PRs

**Review by thiennn — CHANGES_REQUESTED** [review]
